frontend/talentsearch/src/js/components/diversityEquityInclusion/EquityOptions.tsx

```diff
@@ -89,50 +90,40 @@ const EquityOptions: React.FC<EquityOptionsProps> = ({
             </h3>
             {hasItems ? (
               <div data-h2-display="b(flex)" data-h2-flex-direction="b(column)">
-                {resolvedDisability && (
+                {resolvedWoman && (
                   <EquityOption
                     isAdded
-                    onOpen={() => setDisabilityOpen(true)}
-                    title={intl.formatMessage({
-                      defaultMessage:
-                        '"I Identify as a person with a disability"',
-                      description:
-                        "Title for when someone indicates they have a disability on their profile",
-                    })}
+                    onOpen={() => setWomanOpen(true)}
+                    title={intl.formatMessage(
+                      getEmploymentEquityStatement("woman"),
+                    )}
```

Comment:
More a question than a request, but should we also make this change for the unresolved items? Or at least move them to `localizedConstants.tsx`?

Reply:
Good catch. I think this is out of scope though and have created a new issue:

- https://github.com/GCTC-NTGC/gc-digital-talent/issues/3484